components/movementsensor/imuwit/imu.go

```diff
@@ -204,13 +205,28 @@ func NewWit(
 	utils.PanicCapturingGo(func() {
 		defer utils.UncheckedErrorFunc(port.Close)
 		defer i.activeBackgroundWorkers.Done()
-
+		timer := time.NewTicker(time.Duration(100 * float64(time.Millisecond)))
```

Comment:
the logic from L. 204 in this function to the end is now doing enough that it warrants its own function, maybe called `startUpdateLoop` or something similar?

Reply:
done